### PR TITLE
Login

### PR DESCRIPTION
**Pull request by walahamad**

### CODE REVIEW

File: src/com/generic/util/SelectorUtil.java [hunk 1]
```diff
@@ -979,4 +979,51 @@ public static WebElement getRandomWebElement(List<WebElement> items) throws Exce
...
+	/**
+	* Check if GWT Loader.
+	*
+	* @return boolean
+	* @throws Exception
+	*/
+	public static void waitGWTLoadedEventPWA() throws Exception {
+		getCurrentFunctionName(true);
+		boolean gwtLoadedEventPWA = CheckGWTLoadedEventPWA();
+		while (!gwtLoadedEventPWA) {
```
Comment (ibatta) [inline src/com/generic/util/SelectorUtil.java]: Please check this formate 
int tries = 0 ; 
while (!CheckGWTLoadedEventPWA()){
Thread.sleep(500);
if(tries == 9)
{
thow new nosuch elemnet exception (); 
}
tries ++; 
}

Comment (walahamad) [reply src/com/generic/util/SelectorUtil.java]: @ibatta  Done

File: GR/com/generic/tests/GR/login/LoginRegression.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
-<suite name="Login Regression" parallel="tests" thread-count="1" data-provider-thread-count="1">
+<suite name="Login Regression" parallel="tests" thread-count="1" data-provider-thread-count="3">
```
Comment (ibatta) [inline GR/com/generic/tests/GR/login/LoginRegression.xml]: thread-count="3"
data-provider-thread-count="1"

Comment (walahamad) [reply GR/com/generic/tests/GR/login/LoginRegression.xml]: @ibatta Changed on pull request #113

**Pull request merged**